### DISCUSSION TITLE
Is there a way to include plugins in qutebrowser?

### DISCUSSION THREAD

**Discussion by g0t0wasd** [qutebrowser/qutebrowser#5825] [Ideas] [answered]
I really love the qutebrowser, but there is some functionality that I need in order to use it as a daily driver. In most cases this functionality is provided by plugins  and if something could be replaced with external tools (like speedup via using mpv or passFF via dmenu) some plugins are not (i.e. uBlock Origin, Privacy Badger, HTTPSEverywhere). Recently I checked qutebrowser with privacytools.io and the results are not satisfying. So I wanted to ask: are there any plans to include support for plugin system? Maybe there is some tutorial on how to write and **distribute** the most basic on your own? If this is impossible what are the blockers for this?

**Comment by rien333** [comment, marked as answer]
See https://github.com/qutebrowser/qutebrowser/issues/30 (tldr; No, but plugin support has been long in the works).
